weather_data_fetcher.py

```diff
@@ -81,6 +69,25 @@ def fetch_weather_data(api_key: str, location_coords: Tuple[int, int]) -> None:
     print(output)
 
 
+def fetch_new_data(api_key: str, location_coords: Tuple[float, float]) -> json:
```

Comment:
You have two choices: 

- Make the function an internal function by starting with an underscore `_`
- Write a more complete docstring describing the function including the parameters and return value